weave-js/vite.config.ts

```diff
@@ -138,7 +138,14 @@ export default defineConfig(({mode, command}) => {
           secure: false,
           changeOrigin: true,
         },
-        '^.*/__frontend/.*': {
+        // Dynamically generated env.js
+        '^.*/__frontend/env.js': {
```

Comment:
Wouldn't this be captured by the regex you inserted below?

Reply:
it would be, but i kind of like the explicit nature of env as it does not exist on disk

Reply:
sg